MODULE.bazel

```diff
@@ -12,9 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(
+    name = "protovalidate",
+    version = "0.10.0",
```

Comment:
How can we automate setting this value so cutting releases does not need to remember to set it?

Reply:
Per discussion: Let's restrict release tagging to a release workflow, and have it do the work of ensuring that the module version is consistent with the tag.